terraform/fec_rds.tf

```diff
@@ -93,13 +93,21 @@ resource "aws_db_instance" "rds_production" {
   publicly_accessible = true
   storage_encrypted = true
   multi_az = true
+  auto_minor_version_upgrade = true
+  storage_type = "io1"
+  identifier = "fec-govcloud-prod"
```

Comment:
I have a slight preference to changing the names to something more friendly, I think it might help reduce human error that might happen from thinking you are a different rds box than you are.